Makefile

```diff
@@ -11,7 +11,8 @@ EXE=
 
 BOOKELEMS= openssl-book.tex
 
-all: openssl-book.pdf
+all: openssl-book.pdf \
+	tls/understand-tls/understand-tls.tex
```

Comment:
Hmm?  Shouldn't it rather be added in `BOOKELEMS` above?  And `foundations/about/about.tex` should as well, yeah?

Reply:
Oh, yeah. You're right - good catch. Fixed.